apps/onnx/model.cpp

```diff
@@ -2,483 +2,573 @@
 #include <pybind11/pybind11.h>
 #include <pybind11/stl.h>
 
-#include "common_types.h"
-#include "onnx_converter.h"
+#include <sys/time.h>
```

Comment:
Pretty sure this won't compile on MSVC builds

Reply:
I'll eventually switch to std::chrono::high_resolution_clock but if you don't mind and since the problem exists in the merged code I'd rather handle this in a separate PR.

Reply:
SGTM